react/features/filmstrip/components/web/ConnectionStatisticsTable.js

```diff
@@ -0,0 +1,499 @@
+import React, { Component } from 'react';
+
+import { translate } from '../../../base/i18n';
+
+/**
+ * React {@code Component} for displaying connection statistics.
+ *
+ * @extends Component
+ */
+class ConnectionStatisticsTable extends Component {
+    /**
+     * {@code ConnectionStatisticsTable} component's property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * Statistics related to bandwidth.
+         * {{
+         *     download: Number,
+         *     upload: Number
+         * }}
+         */
+        bandwidth: React.PropTypes.object,
+
+        /**
+         * Statistics related to bitrate.
+         * {{
+         *     download: Number,
+         *     upload: Number
+         * }}
+         */
+        bitrate: React.PropTypes.object,
+
+        /**
+         * Statistics related to framerates for each ssrc.
+         * {{
+         *     [ ssrc ]: Number
+         * }}
+         */
+        framerate: React.PropTypes.object,
+
+        /**
+         * Whether or not the statitics are for local video.
+         */
+        isLocalVideo: React.PropTypes.bool,
+
+        /**
+         * Callback to invoke when the show additional stats link is clicked.
+         */
+        onShowMore: React.PropTypes.func,
+
+        /**
+         * Statistics related to packet loss.
+         * {{
+         *     download: Number,
+         *     upload: Number
+         * }}
+         */
+        packetLoss: React.PropTypes.object,
+
+        /**
+         * Statistics related to display resolutions for each ssrc.
+         * {{
+         *     [ ssrc ]: {
+         *         height: Number,
+         *         width: Number
+         *     }
+         * }}
+         */
+        resolution: React.PropTypes.object,
+
+        /**
+         * Whether or not additional stats about bandwidth and transport should
+         * be displayed. Will not display even if true for remote participants.
+         */
+        shouldShowMore: React.PropTypes.bool,
+
+        /**
+         * Invoked to obtain translated strings.
+         */
+        t: React.PropTypes.func,
+
+        /**
+         * Statistics related to transports.
+         */
+        transport: React.PropTypes.array
+    };
+
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     * @returns {ReactElement}
+     */
+    render() {
+        const { isLocalVideo } = this.props;
+
+        return (
+            <div className = 'connection-info'>
+                { this._renderStatistics() }
+                { isLocalVideo ? this._renderShowMoreLink() : null }
+                { isLocalVideo && this.props.shouldShowMore
```

Comment:
The only catch there is in the current jitsi-meet if you click show more and then close the popover and re-open it, it will automatically have the stats expanded because the show more state is saved in the connection indicator. If the show more state is put into the component state, then on re-open it won't have stats expanded. I would like to make the change, because it would get rid of an extra bit of state, but only if the ux change is okay.